api/src/main/java/javax/enterprise/inject/spi/ProcessBeanAttributes.java

```diff
@@ -92,4 +92,16 @@
      * @throws IllegalStateException if called outside of the observer method invocation
      */
     public void veto();
+
+
+    /**
+     * Forces bean creation even if the bean class has non static and non private final methods.
```

Comment:
This wording is imho inaccurate. First of all, the container must only validate that a bean type is proxyable if an injection point resolves to a bean that requires a client proxy or has interceptor or has decorator. See also 3.15. Unproxyable bean types.

Also `non-static, final methods with public, protected or default visibility` would be probably better than ` non static and non private`.